en/complete_vehicles/atl_mantis_edu.md

```diff
@@ -0,0 +1,139 @@
+# ATL Mantis EDU
+
+:::warning
+The ATL Mantis EDU is currently work in progress. Use with caution.
+:::
+
+The ATL Mantis EDU is based on the ready-to-fly Yuneec Mantis G and geared towards education.
+
+## Purchase
+
+The ATL Mantis EDU version will be released soon.
+TODO: add purchase link
+
+## Hardware overview
+
+The Mantis broadly features the following hardware components:
+- Autopilot/FMU based on STM32F7.
+- Gimbal controller.
+- Camera system on a chip with WiFi (forwards MAVLink messages from autopilot and gimbal via Wifi)
+- Downward-facing flow module with laser range sensor.
+
+## Update components
+
+The files to update the Mantis can currently be found on [GoogleDrive](https://drive.google.com/drive/u/1/folders/1tDo9AktWtB47LRzaRaMxDkONTNQHsjCG).
+
+### Update camera
```

Comment:
FYI, the rest of the user guide uses first-letter heading capitalization, rather than sentence capitalisation. It doesn't matter particularly as long as you are consistent in a document.